examples/fabric-bridge-app/linux/main.cpp

```diff
@@ -34,8 +35,9 @@
 #include <sys/ioctl.h>
 #include <thread>
 
-using namespace chip;
+void MatterEcosystemInformationPluginServerInitCallback();
```

Comment:
where does this actually get declared? Why not just include the header? Is this one of those fancy ember things where you have to do weird stuff to swap out impls at build?

Reply:
This is one of those fancy embr/zap things where the header only gets created if you have a static endpoint for ECOINFO. ECOINFO is only a dynamic endpoint cluster so that doesn't get auto generated for us

https://github.com/project-chip/connectedhomeip/blob/master/scripts/py_matter_idl/matter_idl/generators/cpp/application/PluginApplicationCallbacksHeader.jinja